ase/src/main/java/com/ase/ase/rest/controller/BasicCovidController.java

```diff
@@ -38,7 +36,23 @@
                 new HospitalSituationPerDateDto(provinceId, Arrays.asList(
                         new HospitalSituationPerDate("10.10.2020", 12, 10),
                         new HospitalSituationPerDate("12.10.2020", 12, 12))));
+
     }
 
+    @CrossOrigin
+    @GetMapping(value = "/tableData/{province-id}", produces = "application/json")
+    @ResponseBody
+    public ResponseEntity<TableDataPerProvinceDto> listNewTableDataFor(@PathVariable("province-id") int provinceId) {
+        return ResponseEntity.ok(
+                new TableDataPerProvinceDto(provinceId, Arrays.asList(
+                        new TableDataPerDate("10.10.2020", 12, 10, Arrays.asList(
+                                new HospitalSituationPerDate("10.10.2020", 12, 10),
```

Comment:
Later the Hospital data should be transmitted as sum from the SQL statement, then we do not need a List

Reply:
The controller should return the data in that format that is expected, i.e., the sum per date. The DB method is missing which is okay but the dummy data should have the correct format. You should not implement the frontend dependent on a data structure when you know that it will change. Use the right structure from the beginning.